scripts/check-structure.js

```
@@ -0,0 +1,417 @@
+        ...fixCommands,
+        '',
+        'echo "结构修复完成！"',
+        'echo "请运行 node scripts/check-structure.js 验证修复结果"'
+      ].join('\n');
+
+      fs.writeFileSync(scriptPath, scriptContent);
+      execSync(`chmod +x ${scriptPath}`);
+      
+      this.log(`\n✅ 修复脚本已生成: ${scriptPath}`, 'success');
+      this.log('运行修复脚本: ./scripts/fix-structure.sh', 'info');
+    } else {
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)